lib/shared_widgets/nav_bar_widget.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class NavBar extends StatelessWidget {
+  const NavBar({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return BottomNavigationBar(
```

Comment:
var strings = AppLocalizations.of(context)!;

Reply:
done